src/documents/views/ReviewDetailView.js

```diff
@@ -0,0 +1,223 @@
+          icon="warning circle"
+          header="Error"
+          content={error.message}
+        />
+      </Container>
+    );
+
+  if (!allowViewReview)
+    return (
+      <Container as={Segment} basic>
+        <Helmet>
+          <title>
+            {`KF Data Tracker - Study review detail - Error ${
+              study ? 'for ' + study.kfId : null
+            }`}
+          </title>
+        </Helmet>
+        <Message
+          negative
+          icon="warning circle"
+          header="Not allowed"
+          content="You don't have access to view this data review."
+        />
+      </Container>
+    );
+  return (
+    <Grid as={Segment} basic container columns={1}>
+      <Helmet>
+        <title>
+          {`KF Data Tracker - Study review detail ${
+            study ? 'for ' + study.name : null
+          }`}
+        </title>
+      </Helmet>
+      {review && (
+        <ReviewDetail
+          reviewNode={review}
+          updateReview={allowChangeReview && updateReview}
+          awaitReview={awaitReview}
+          approveReview={approveReview}
+          closeReview={closeReview}
+          reopenReview={reopenReview}
+          updateError={updateError}
+          awaitError={awaitError}
+          approveError={approveError}
+          closeError={closeError}
+          reopenError={reopenError}
+          allReviews={allReviews}
+          setAddFileOpen={setAddFileOpen}
+          downloadFileMutation={downloadFileMutation}
+          canAddFile={
+            files.filter(({node}) => !addedFiles.includes(node.kfId)).length >
+              0 &&
+            review.state !== 'completed' &&
+            review.state !== 'closed' &&
+            allowChangeReview
+          }
+        />
+      )}
+      <Modal open={addFileOpen}>
+        <Modal.Header>Add documents to current data review</Modal.Header>
+        <Modal.Content>
+          <ReviewFileSection
+            files={files.filter(({node}) => !addedFiles.includes(node.kfId))}
+            studyId={kfId}
+            selectedFiles={selectedFiles}
+            setSelectedFiles={setSelectedFiles}
+          />
+        </Modal.Content>
+        <Modal.Actions>
+          {addFileError && <span className="text-red">{addFileError}</span>}
+          <Button
+            onClick={() => {
+              setAddFileOpen(false);
+            }}
+          >
+            Cancel
+          </Button>
+          <Button
+            primary
+            disabled={selectedFiles.length === 0 || addFileError.length > 0}
+            onClick={() => {
+              const addedVersions = review
+                ? review.versions.edges.map(({node}) => node.id)
+                : [];
+              updateReview({
+                variables: {
+                  id: review.id,
+                  input: {
+                    name: review.name,
+                    description: review.descrition,
+                    versions: addedVersions.concat(selectedFiles),
```

Comment:
Sometimes this adds the same document twice.
I've noticed this specifically happens when you add one document, then open the modal again to add another. It will try to post the first document twice. Maybe some state that doesn't get flushed for the selection?